src/test/java/uk/gov/companieshouse/psc/extensions/api/controller/impl/ValidationStatusControllerImplTest.java

```diff
@@ -30,10 +30,10 @@
 
 @ExtendWith(MockitoExtension.class)
 class ValidationStatusControllerImplTest {
-    private final String TRANSACTION_ID = "transaction-id";
-    private final String FILING_RESOURCE_ID = "filing-resource-id";
-    private final String PSC_NOTIFICATION_ID = "psc-notification-id";
-    private final String COMPANY_NUMBER = "12345678";
+    private final String transactionId = "transaction-id";
```

Comment:
depending on what you decide for this in test above, whether to make this static